modin/pandas/base.py

```diff
@@ -2864,6 +2864,32 @@ def tz_localize(
         )
         return self.set_axis(labels=new_labels, axis=axis, inplace=not copy)
 
+    # TODO: uncomment the following lines when #3331 issue will be closed
+    # @prepend_to_notes(
+    #     """
+    #     In comparison with pandas, Modin's ``value_counts`` returns Series with ``MultiIndex``
```

Comment:
Why do we have this behavior?

Reply:
answer is [here](https://github.com/modin-project/modin/pull/2730#discussion_r700153986)

Reply:
I am starting to think that we need to have some additional "compatibility mode" option in Modin that would try to reproduce the closest possible to Pandas behavior even at an expense of performance. For example #3298 also introduces a known difference, but in that case it is very subtle. In this case an object of different class (Index instead of MultiIndex) may really break some user's application when they replace `import pandas` with `import modin.pandas`. 

What do you think if adding some `MODIN_FULL_PANDAS_COMPAT` option to enable workarounds for such differences?

Possibly this shouldn't be discussed in this PRs comments and brought somewhere outside.